### PR TITLE
Issue 441: rslurm example updates

### PR DESCRIPTION
**Pull request by atombaby**
Added a much simpler example for rslurm

### CODE REVIEW

**Comment by vortexing** [comment]
Is this ready for @jennylsmith to weigh in on it?  Sorry to volunteer you Jenny.

File: _compdemos/rslurm_intro.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+
+## Using rslurm
+
+### Requirements
+
+You will need access to the Rhinos for submitting the job 
+
+### Instructions on how to get set-up
+
+Log into a rhino or in an NoMachine terminal session.  Use the module commands to load R- most later versions of R in our environment have the `rslurm` module ainstalled but it is also available for installation via CRAN
```
Comment (jennylsmith) [inline _compdemos/rslurm_intro.md:30]: Could add a link to NoMachine (https://sciwiki.fredhutch.org/scicomputing/access_methods/#nomachine-nx-multi-os). If the reader hasn't heard of it, they may be interested in learning more.

File: _compdemos/rslurm_intro.md [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+The two primary functions for running jobs are `slurm_apply` and `slurm_call`, each of which create Slurm jobs.  The both take an R function to be run and the parameters for that function as arguments.  `slurm_apply` takes a set of multiple parameters whereas `slurm_call` takes only a single set of parameters.
+
+## Workflow
+
+The first step in using `rslurm` for parallelizing a computation is to create the function.  The function will get its arguments from the `slurm_apply` function.  For example, in our example of calculating &pi; we need a function that generates a random set of coordinates and determines which of those coordinates are inside a circle.
+
+```
+sim.pi <- function(iterations = 1000) {
+  # Generate two vectors for random points in unit 
+  # circle x.pos <- runif(iterations, min=-1, max=1)
```
Comment (jennylsmith) [inline _compdemos/rslurm_intro.md:50]: Looks like the `x.pos` variable got commented out here.

File: _compdemos/rslurm_intro.md [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+  cpus_per_node=1
+)
+```
+
+This creates a job object (`sjob1`).  The arguments for this function are:
+
+ - sim.pi - the function to be run in each Slurm job
+ - params - a data frame containing the parameters for each call of the function
+ - jobname - the name the jobs will have in Slurm
+ - nodes - this is misleading- this is more correctly called "tasks" for Slurm The rows in the parameters will be distributed over these tasks
```
Comment (jennylsmith) [inline _compdemos/rslurm_intro.md:88]: Just needs a period after Slurm.

**Comment by vortexing** [comment]
Once these comments are resolved, we can merge!!!

**Comment by vortexing** [comment]
I think this is in @atombaby 's court to decide if Jenny's last comments are incorporated or not.  Then I can merge it!

**Pull request merged**